tests/e2e/configurer/chain/commands.go

```diff
@@ -73,14 +73,18 @@ func (n *NodeConfig) CreateConcentratedPool(from, denom1, denom2 string, tickSpa
 func (n *NodeConfig) CreateConcentratedPosition(from, lowerTick, upperTick string, token0, token1 string, token0MinAmt, token1MinAmt int64, freezeDuration string, poolId uint64) uint64 {
 	n.LogActionF("creating concentrated position")
 
-	cmd := []string{"osmosisd", "tx", "concentratedliquidity", "create-position", lowerTick, upperTick, token0, token1, fmt.Sprintf("%d", token0MinAmt), fmt.Sprintf("%d", token1MinAmt), freezeDuration, fmt.Sprintf("--from=%s", from), fmt.Sprintf("--pool-id=%d", poolId)}
-	outBuf, _, err := n.containerManager.ExecTxCmd(n.t, n.chainId, n.Name, cmd)
+	cmd := []string{"osmosisd", "tx", "concentratedliquidity", "create-position", lowerTick, upperTick, token0, token1, fmt.Sprintf("%d", token0MinAmt), fmt.Sprintf("%d", token1MinAmt), freezeDuration, fmt.Sprintf("--from=%s", from), fmt.Sprintf("--pool-id=%d", poolId), "-o json"}
+	outJson, _, err := n.containerManager.ExecTxCmdWithSuccessString(n.t, n.chainId, n.Name, cmd, "code\":0")
 	require.NoError(n.t, err)
 
-	prefix := "position_id\",\"value\":\""
-	startIndex := strings.Index(outBuf.String(), prefix) + len(prefix)
-	endIndex := startIndex + strings.Index(outBuf.String()[startIndex:], "\"}")
-	positionID, err := strconv.ParseUint(outBuf.String()[startIndex:endIndex], 10, 64)
+	var txResponse map[string]interface{}
```

Comment:
What if we define a struct with all json fields specified for parsing, like:

```go
type txResponse struct {
    events `json:"events,omitempty"`
   ...
}
```

this might help reduce the need for json parsing and let `json.Unmarshal` handle it

Reply:
I think we can do this in a follow on issue. I will merge this to stop CI from failing 50 percent of the time